OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -157,10 +159,23 @@ void SelectMod(ModMetadata mod)
 				modOffset = selectedIndex - 4;
 
 			loadButton.Text = modInstallStatus[mod] ? "Load Mod" : "Install Assets";
+
```

Comment:
I think it would be cleaner to combine these two together into a single loadButton.Text assignment.